src/smvlang/smv_language.cpp

```diff
@@ -168,7 +172,9 @@ void smv_languaget::show_parse(std::ostream &out, message_handlert &)
       if(it->second.type.id()=="submodule")
       {
         std::string msg;
-        type2smv(it->second.type, msg);
+        symbol_tablet symbol_table;
+        namespacet ns(symbol_table);
```

Comment:
Do we really not have a symbol table or namespace available when invoking `show_parse`?

Reply:
Yes, this is pre typechecking.